mmv1/templates/terraform/pre_delete/spanner_instance.go.erb

```diff
@@ -0,0 +1,25 @@
+
+if d.Get("force_destroy").(bool) {
+	// Override GET url to get backups
+	url, err = replaceVars(d, config, "{{SpannerBasePath}}projects/{{project}}/instances/{{name}}/backups")
+	if err != nil {
+		return err
+	}
+
+	resp, err := sendRequest(config, "GET", billingProject, url, userAgent, nil)
+	if err != nil {
+		// API returns 200 if no backups exist but the instance still exists. This error is gua
```

Comment:
nit: did this text get clipped?

Reply:
last sentence isn't supposed to be there. Fixed